src/webpimage.cpp

```diff
@@ -522,7 +522,7 @@ namespace Exiv2 {
             const uint32_t size_u32 = Exiv2::getULong(size_buff, littleEndian);
 
             // Check that `size_u32` is safe to cast to `long`.
-            enforce(size_u32 <= static_cast<size_t>(std::numeric_limits<long>::max()),
+            enforce(static_cast<size_t>(size_u32) <= static_cast<size_t>(std::numeric_limits<long>::max()),
```

Comment:
Wrap the enforce in a 
```C++
#if defined(_WIN32) || defined(__CYGWIN__) || defined(__MINGW__) || defined(_MSC_VER)
enforce(...);
#endif
```

Reply:
Thanks, I'll do that if I cannot defeat the compiler warning. I have one more thing that I will try first.